tests/test.py

```diff
@@ -31,10 +31,18 @@ def upload_jar(jar):
     return jar_url
 
 
-def submit_job(jar_url):
-    spark_job_runner_args = 'http://leader.mesos:5050 dcos \\"*\\" spark:only 2'
-    submit_args = "-Dspark.driver.memory=2g --class com.typesafe.spark.test.mesos.framework.runners.SparkJobRunner {0} {1}".format(
-        jar_url, spark_job_runner_args)
+def submit_job(app_class, python_lib_url, app_resource_url, app_args):
+    if (app_class):
```

Comment:
if these parameters are options, let's make them keyward parameters which default to None.  Also, do `if app_class is not None`